dotbot/protocol.py

```diff
@@ -65,9 +66,9 @@ class PacketType(IntEnum):
 
     BEACON = 1
     JOIN_REQUEST = 2
-    JOIN_RESPONSE = 3
-    LEAVE = 4
-    DATA = 5
+    JOIN_RESPONSE = 4
+    LEAVE = 8
```

Comment:
Note that instead of LEAVE, now we have KEEP_ALIVE

Reply:
Thanks, will fix that!

Reply:
Done